app/gui/view/debug_scene/cloud-dashboard/src/lib.rs

```diff
@@ -37,131 +37,483 @@
+                    grid.model_for_entry_needed.filter_map(f!((input) model.model_for_entry(input)));
+            }
+
+
+            // === Configure Simple Grid View ===
+
+            grid.set_entries_size(Vector2(130.0, ENTRY_HEIGHT));
+            let params = ensogl_grid_view::simple::EntryParams {
+                // FIXME [NP]: Turn these into StyleWatchFrp values or at the very least constants.
+                bg_color: color::Lcha::transparent(),
+                bg_margin: 1.0,
+                // FIXME [NP]: Apply the hover color to the whole row, not just the selected entry.
+                hover_color: color::Lcha::from(color::Rgba(
+                    62f32 / u8::MAX as f32,
+                    81f32 / u8::MAX as f32,
+                    95f32 / u8::MAX as f32,
+                    0.05,
+                )),
+                selection_color: color::Lcha::from(color::Rgba(1.0, 0.0, 0.0, 1.0)),
+                ..default()
+            };
+            grid.set_entries_params(params);
+            let row = model.rows();
+            let col = Columns::LEN;
+            grid.reset_entries(row, col);
+            grid.frp().focus();
+
+
+            // === Configure Simple Grid View Headers ===
+
+            let header_frp = grid.header_frp();
+            frp::extend! { network
+                requested_section <- header_frp.section_info_needed.map(|&(row, col)| {
+                    let sections_size = 2 + col;
+                    let section_start = row - (row % sections_size);
+                    let section_end = section_start + sections_size;
+                    let model = header_entry_model(section_start, col);
+                    (section_start..section_end, col, model)
+                });
+                header_frp.section_info <+ requested_section;
             }
+            std::mem::forget(network);
+
+
+            // === Configure Scrollable Grid View ===
 
-            View { frp, model }
+            // These sizes are arbitrary, because the grid will get resized as soon as the WebGL
+            // context is ready (because the shape of the viewport will change as a result).
+            grid.scroll_frp().resize(Vector2(400.0, VIEWPORT_HEIGHT));
+
+            app.display.add_child(&model.root);
+
+
+            // === Populate Table With Data ===
+
+            let client =
+                enso_cloud_http::Client::new(API_GATEWAY_ID, AWS_REGION, TOKEN.to_string());
+            get_projects(client, input.clone_ref());
+
+
+            std::mem::forget(grid);
+            std::mem::forget(input);
+            std::mem::forget(out);
+
+            Self { frp, model }
         }
     }
 
+    // FIXME [NP]: Should we be assuming that the `row, col` are always in bounds? If we do so, we
+    // can avoid having to return `Option` here, but we run the risk of panicking in the event of an
+    // out of bounds `row, col`. Though this should only happen on implementation error.
```

Comment:
Ask Adam

Reply:
@farmaazon should this method accept `Position` or `Option<Position>`? With the latter, I could return `None` rather than `panic!` but I don't know if we should be receiving incorrect entry requests here in the first place.